apps/labs/posts/announcing-labs-annual-report-2022.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Quansight Labs 2022 Annual Report: Celebrating Growth and Sustainability'
+author: pavithra-eswaramoorthy
+published: February 7, 2023
+description: 'Presenting our first annual report! Read about our project achievements, community initiatives, and work culture.'
+category: [Community, PyData ecosystem]
+featuredImage:
+  src: /posts/hello-world-post/featured.png
+  alt: 'TODO'
+hero:
+  imageSrc: /posts/hello-world-post/hero.jpeg
+  imageAlt: 'TODO'
+---
+
+_Co-authors: Noa Tamir, Ralf Gommers, and Tania Allard._
+_Designer: Irina Fumarel_
+
+We at Quansight Labs are delighted to share our first-ever annual report with you!
+
+In these pages, we reflect on the past year and share how we're progressing towards our mission to sustain and grow community-driven open source projects and ecosystems.
+
+{
+// TODO: Add Infographic with alt-text
+}
+
+As members of the PyData community, the success of the open source data science ecosystem is our success. In 2022, the projects we contribute to and collaborate with continued to improve and flourish. We secured grants for substantial advancements and essential maintenance work, collaborated with industry partners on key initiatives, focused on accessibility initiatives for more people to benefit from our tools, and welcomed new enthusiastic contributors with diverse skills and backgrounds.
+
+Most of the projects we contribute to are supported by NumFOCUS, a non-profit organization essential for the overall sustainability of our ecosystem. Extending support back to NumFOCUS is vital to our mission, so we financially sponsor NumFOCUS every quarter, corresponding to the hours our team contributes to NumFOCUS sponsored and affiliated projects. In 2022, we're very proud to have sent over $30,000. We're equally proud of the team we built and the culture we fostered at Labs throughout the year, which enabled all of these achievements.
+
+Learn about our projects, our team, and how we work at Labs in the full report below:
+
+[Download Quansight Labs Annual Report 2022](labs.quansight.org/annual-reports/quansight-labs-annual-report-2022.pdf)
```

Comment:
```suggestion
[Download Quansight Labs Annual Report 2022](/annual-reports/quansight-labs-annual-report-2022.pdf)
```

Relative URLs help test on local and staging.

Reply:
thanks, done!